.grunt-config/widgets-css.js

```diff
@@ -131,23 +133,30 @@ class WidgetsCss {
 		fs.readdirSync( filePath ).forEach( ( fileName ) => {
 			const fileFullPath = path.join( filePath, fileName );
 
+			const isFrontendScssFile = fileName.indexOf( 'frontend.scss' ) > -1,
+				isWidgetsScssFile = fileName.indexOf( '.scss' ) > -1 && filePath.indexOf( '/widgets' ) > -1;
```

Comment:
@louiswol94 I have added support to include all files inside the /widgets folder here.

Wdyt?